for var-som-mx8m-plus

breakAIDev opened the pull request:
I have corrected the cmake/thirdparty/openfst*.cmake.

breakAIDev commented:
I have resolved the conflicts.

breakAIDev commented:
I have corrected all code factor issues.

jtrmal commented:
I don't understand the purpose of your changes and as of now, I'm not inclined to merge.

breakAIDev commented:
The current version works well on PCs. So, I tried to build STT using it on NXP CPUs. During the process, I realized that the features I modified were needed in the current version, so I added them. I believe that adding these features will make the current version compatible with more devices.
